k8_cortx_cloud/solution.yaml

```diff
@@ -50,7 +50,20 @@ solution:
         sns: 1+0+0
         dix: 1+0+0
     external_services:
-      type: LoadBalancer
+      s3:
```

Comment:
Can you make these same changes in the solution_stub.yaml as well? This will ensure we can keep them in sync and not have stub deployments fail validation.